backend/src/services/v2/smtp/templates/baseEmailTemplate.ts

```diff
@@ -3,9 +3,14 @@ import dedent from 'dedent-js'
 import { ReviewKindKeys } from '../../../../types/v2/enums.js'
 
 export interface IEmailTemplate {
-  getSubject(resourceName: string): string
-  getHtml(releaseName: string, reviewKind: ReviewKindKeys, modelId: string, baseUrl: string, author: string): string
-  getBody(releaseName: string, reviewKind: ReviewKindKeys, modelId: string, baseUrl: string, author: string): string
+  to
```

Comment:
These should be typed, e.g.  as `string`.

Reply:
I don't think you need this interface for anything.  You can use the `class` to automatically generate these types.  Just add these as abstract functions to your class, and add the types in.  Then you can just use the class and not this interface.  See our v2 authorization class as an example.